src/inference/include/openvino/runtime/config.hpp

```diff
@@ -0,0 +1,616 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
Some personal impression. For me it was quite challenging to read this file and understand which key belongs to which namespace (due to some long inline code and comments).

Reply:
Agreed Any suggestions? Should I split the file into separate files? Or config folder with separate config files?

Reply:
I don't have strong opinion here, let's ask other folks. @ilyachur @ilya-lavrenov ?